### ISSUE TITLE
New PyPI release?

### ISSUE THREAD

**Issue by mariehbourget** [neuropoly/changelog#31]
Currently, the [release instructions](https://github.com/ivadomed/ivadomed/wiki/Releases) of ivadomed mention to use neuropoly-changelog to update CHANGE.md.

However, the behavior is different depending if we use the PyPI Standard install (v1.1.0) or the Development install i.e. some labels are missing from the changelog with v1.1.0 because of PR #28 that was merged after the release of v1.1.0.

I see that another PR (#30) that allows to specify labels on the command line, but that PR is not included in v1.1.0 either.

Is it possible/planned to do a new stable PyPI release of changelog?
Or should I update the ivadomed instructions to only use the Development install with labels in command line?

**Comment by mariehbourget** [comment]
Hi @dyt811, @joshuacwnewton,
Just following-up on this issue with you (I forgot to tag you the first time) It is not urgent at all, but I think it would be worth it to do a new stable release.
Let me know your thoughts, thanks!

**Comment by joshuacwnewton** [comment]
> Is it possible/planned to do a new stable PyPI release of changelog?
> Or should I update the ivadomed instructions to only use the Development install with labels in command line?

Honestly, given that the `changelog` tool is only used internally, and that updates only come about infrequently when there is an internal need, I'm not sure that there is too much benefit in spending the effort to maintain stable public releases. I think it would be perfectly okay to use the development version?

Case in point: SCT is currently using the development version of `changelog` as part of an automated GitHub Actions workflow ([link](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/master/.github/workflows/automate-changelog-PR.yml)) that creates a PR with the updated changelog.

Perhaps it could be useful for Ivadomed to adopt a similar workflow, too?

**Comment by mariehbourget** [comment]
> I think it would be perfectly okay to use the development version?



> Perhaps it could be useful for Ivadomed to adopt a similar workflow, too?

Yes of course! Currently our workflow is entirely manual in ivadomed anyway 😅.
I just found the discrepancy by chance while updating the changelog manually.

If there is no need to maintain the stable version, that's ok with me as well. But I would suggest removing the standard install from the readme [here](https://github.com/neuropoly/changelog#standard-install), so we only and always use the dev install.

**Comment by joshuacwnewton** [comment]
> But I would suggest removing the standard install from the readme [here](https://github.com/neuropoly/changelog#standard-install), so we only and always use the dev install.

Good call! :+1:
